src/nvim/msgpack_rpc/server.c

```diff
@@ -67,6 +68,27 @@ void server_teardown(void)
   GA_DEEP_CLEAR(&watchers, SocketWatcher *, close_socket_watcher);
 }
 
+/// Return unique address for local server.
+///
+/// In Windows this is a local pipe address in the format
+///     \\.\pipe\nvim-<PID>-<COUNTER>.
+///
+/// For other systems it is a path returned by vim_tempname().
+///
+/// This function is NOT thread safe
+char *server_address(void)
```

Comment:
`server_address_create` or `server_address_generate` would be a less ambiguous name.